sycl/source/detail/device_impl.cpp

```diff
@@ -546,6 +546,11 @@ bool device_impl::has(aspect Aspect) const {
             sizeof(pi_bool), &support, nullptr) == PI_SUCCESS;
     return call_successful && support;
   }
+  case aspect::ext_oneapi_non_uniform_groups: {
+    return (this->getBackend() == backend::ext_oneapi_level_zero) ||
```

Comment:
@Pennycook I also wanted to check that this correctly covers all the SPIRV cases: Are 
```
(this->getBackend() == backend::ext_oneapi_level_zero) ||
           (this->getBackend() == backend::opencl)
```
The only two backends supporting the spirv non uniform groups?

Thanks

Reply:
I think these are the only two that we've tested, at least.  I'm happy to leave this as it is for now -- if we can prove to ourselves later that things should run elsewhere, we can update the aspect.